### ISSUE TITLE
edit template for scheduler

### ISSUE THREAD

**Issue by minro** [aurelia-ui-toolkits/aurelia-kendoui-bridge#491]
When trying to create a template for the scheduler edit event dialog, like this:
```
<ak-scheduler ... >
    <ak-template for="editTemplate">
        <div>test</div>
    </ak-template>
    ...
```

I get the following error:
```es6.promise.js:134 Unhandled promise rejection Error: Invalid template property name: "editorTemplate", valid values are: allDayEventTemplate, dateHeaderTemplate, eventTemplate, groupHeaderTemplate, majorTimeHeaderTemplate, minorTimeHeaderTemplate(…)```

The KendoUI way to do this can be found here: [Customize Edit and Event Templates](http://docs.telerik.com/kendo-ui/controls/scheduling/scheduler/how-to/custom-edit-and-event-templates)

**Comment by JeroenVinke** [comment]
I just checked and it looks like the template you can supply for the popup isn't compiled by aurelia's templating system. This process is always triggered by the Kendo control but there are some templates that do not trigger this process, the edit template seems to be one of them. 

About the error: `editTemplate` is not a property on the scheduler itself, it is on the `editable` object. I think we can add support for the following:

```
    <ak-template for="editable.editTemplate">
        <div>some kendo template as aurelia templating system is not supported here</div>
    </ak-template>
```

We have the same requirement for the grid's popup template: http://docs.telerik.com/kendo-ui/api/javascript/ui/grid#configuration-editable.template, where the `template` property is on an object on the grid.

**Comment by charlespockert** [comment]
@JeroenVinke +1 from me - I could actually use this in one of my apps anyway

**Comment by JeroenVinke** [comment]
https://gist.run/?id=cc21a28212df98d442bc2b3c0e0526a1 is how these kinds of templates can be used without this improvement

**Comment by JeroenVinke** [comment]
@charlespockert I think you'll like the implementation: https://github.com/aurelia-ui-toolkits/aurelia-kendoui-bridge/blob/f1c63ddd550ee0e0898d0dffbec7954ffada7331/src/common/template-gatherer.js#L51-L72

**Comment by JeroenVinke** [comment]
released in 0.10.4:

```
        <ak-template for="editable.template">
          <h3>Edit template</h3>
        </ak-template>
```
